bitnami/jaeger/values.yaml

```diff
@@ -0,0 +1,1035 @@
+## @param fullnameOverride String to fully override common.names.fullname
+##
+fullnameOverride: ""
+## @param kubeVersion Force target Kubernetes version (using Helm capabilities if not set)
+##
+kubeVersion: ""
+## @param commonLabels Labels to add to all deployed objects (sub-charts are not considered)
+##
+commonLabels: {}
+## @param commonAnnotations Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+## Enable diagnostic mode in the deployment
+##
+diagnosticMode:
+  ## @param diagnosticMode.enabled Enable diagnostic mode (all probes will be disabled and the command will be overridden)
+  ##
+  enabled: false
+  ## @param diagnosticMode.command Command to override all containers in the deployment
+  ##
+  command:
+    - sleep
+  ## @param diagnosticMode.args Args to override all containers in the deployment
+  ##
+  args:
+    - infinity
+
+## @section Jaeger parameters
+##
+
+## Bitnami Jaeger image
+## ref: https://hub.docker.com/r/bitnami/jaeger/tags/
+## @param image.registry Jaeger image registry
+## @param image.repository Jaeger image repository
+## @param image.tag Jaeger image tag (immutable tags are recommended)
+## @param image.digest Jaeger image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+## @param image.pullPolicy image pull policy
+## @param image.pullSecrets Jaeger image pull secrets
+## @param image.debug Enable image debug mode
+##
+image:
+  registry: docker.io
+  repository: bitnami/jaeger
+  tag: 1.39.0-debian-11-r4
+  digest: ""
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: https://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+  ## Enable debug mode
+  ##
+  debug: false
+
+## @section Query deployment parameters
+##
+
+query:
+  ## @param query.command Command for running the container (set to default if not set). Use array form
+  ##
+  command: []
+  ## @param query.args Args for running the container (set to default if not set). Use array form
+  ##
+  args: []
+  ## @param query.lifecycleHooks [object] Override default etcd container hooks
+  ##
+  lifecycleHooks: {}
+  ## @param query.extraEnvVars Extra environment variables to be set on jaeger container
+  ## For example:
+  ##  - name: FOO
+  ##    value: BAR
+  ##
+  extraEnvVars: []
+  ## @param query.extraEnvVarsCM Name of existing ConfigMap containing extra env vars
+  ##
+  extraEnvVarsCM: ""
+  ## @param query.extraEnvVarsSecret Name of existing Secret containing extra env vars
+  ##
+  extraEnvVarsSecret: ""
+  ## @param query.replicaCount Number of Jaeger replicas
+  ##
```

Comment:
Could you add these references to values.yaml

```
Key backup.uploadProviders.azure.image not found
Key backup.uploadProviders.google.image not found
Key cassandra.containerPorts.cql not found
Key cassandra.dbUser.password not found
Key cassandra.dbUser.user not found
Key cassandra.image.registry not found
Key cassandra.image.repository not found
Key cassandra.image.tag not found
Key jaeger.initdbScriptsCM not found
Key jaeger.initdbScriptsSecret not found
Key volumePermissions.image not found
```

They are used in the template but not mentioned here. For consistency we want all references done in the templates as values in this file

Reply:
None of that values are now used, except 

```
Key cassandra.service.ports.cql not found
Key cassandra.dbUser.password not found
Key cassandra.dbUser.user not found
```

Which are now declared